apps/settings/js/modules/simpin.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * This is a simple model that can help us mimic the status change event
+ * when enabling / disabling simcard.
+ *
+ * @module SimPin
+ */
+define(function(require) {
+  'use strict';
+
+  var Module = require('modules/base/module');
+  var SettingsUtils = require('modules/settings_utils');
+  var EventEmitter = require('modules/base/event_emitter');
+
+  const EVENT_NAME = 'simpin-';
+  const APIS = [
+    'unlockCardLock',
+    'setCardLock',
+    'updateContact',
+    'getCardLock',
+    'getCardLockRetryCount'
+  ];
+
+  var SimPin = Module.create(function() {
```

Comment:
Maybe we should call this module `SimSecurity` as it handles not only 'sim pin'.